DashKitUI/Views/DashSettingsView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  DashSettingsView.swift
+//  ViewDev
+//
+//  Created by Pete Schwamb on 3/8/20.
+//  Copyright © 2020 Pete Schwamb. All rights reserved.
+//
+
+import SwiftUI
+import LoopKitUI
+
+struct DashSettingsView<Model>: View where Model: DashSettingsViewModelProtocol  {
+    
+    @ObservedObject var viewModel: Model
+
+    weak var navigator: DashUINavigator?
+    
+    var body: some View {
+        List {
+            VStack(alignment: .leading) {
+                VStack(alignment: .center) {
+                    Image(frameworkImage: "Pod")
+                        .resizable()
+                        .aspectRatio(contentMode: ContentMode.fit)
+                        .frame(height: 100)
+                        .padding([.top,.horizontal])
+                }.frame(maxWidth: .infinity)
+
+                HStack(alignment: .lastTextBaseline, spacing: 3) {
+                    Text(self.viewModel.lifeState.localizedLabelText)
+                        .foregroundColor(Color(UIColor.secondaryLabel))
+                    Spacer()
+                    Text("3").font(.title).fontWeight(.bold)
```

Comment:
[nit] I was curious about this "3" constant.  What does this represent?  Not a blocker, just a question.

Reply:
This is a design placeholder. The design is still in progress, and there is a separate ticket for implementing that design.